GVRf/Framework/src/org/gearvrf/GVRScript.java

```diff
@@ -303,4 +313,25 @@ SplashScreen createSplashScreen(GVRViewManager viewManager) {
         onSplashScreenCreated(splashScreen);
         return splashScreen;
     }
+
+    public void addPlugin(GVRPlugin plugin) {
```

Comment:
@dattanand Do you think there is another way of adding in the plugin? Do you think we can only override onInit() to perform needed initialization?